saleor/graphql/app/tests/queries/test_app_extension.py

```diff
@@ -412,8 +428,10 @@ def test_app_extension_with_app_query_by_staff_with_permissions(
 
     # when
     response = staff_api_client.post_graphql(
-        QUERY_APP_EXTENSION_WITH_APP, variables, permissions=[permission_manage_apps]
+        QUERY_APP_EXTENSION_WITH_APP, variables, permissions=[]
```

Comment:
I don't get this open. I actually want to check if user with manage_apps can access the page (so there is no regression with previous behavior) - but test fails with "assert no permissions"